pyartcd/pyartcd/pipelines/update_golang.py

```diff
@@ -251,6 +215,52 @@ async def run(self):
             dry_run=self.dry_run,
         )
 
+    async def process_build(self, el_v, nvr):
+        if await is_latest_and_available(self.ocp_version, el_v, nvr, self.koji_session):
+            return False
+        if not self.tag_builds:
+            return True
+        # Tag builds into override tag
+        self.tag_builds(el_v, nvr)
+        # Regen repo
+        if await self.regen_repo_with_complete(el_v):
+            _LOGGER.info("Regen-repo not success")
+            return False
+        # Wait for repo to be available (5 hours max)
```

Comment:
let's check for dry-run and exit early here

Reply:
updated